ramanujan/cmf/cmf_test.py

```diff
@@ -77,3 +79,14 @@ def test_substitute_trajectory_walk_equivalence():
     assert subbed.walk({n: 1}, iterations, {n: 1}) == unsubbed.walk(
         trajectory, iterations, start
     )
+
+
+def test_cache_calculation_method():
+    cmf = known_cmfs.zeta3()
+    cmf.reset_potential_cache([40,40], initial_mat=Matrix(sp.eye(2)))
+    assert cmf.walk({x:1, y:1}, 40).limit(Vector.zero()) == cmf[40,40].limit(Vector.zero())
```

Comment:
please add some comment about the difference between using "walk" and "[40,40]". It is not clear from the outside that one uses cache and the other not (or which one)